controllers/sriov-controller/Dockerfile.client

```diff
@@ -0,0 +1,27 @@
+FROM centos:latest
+
+#
+# These packages are optional and used only for debugging
```

Comment:
So is this not a production Docker container? The comment above leads me to believe it's more of a debug, pre-production type of thing.

Reply:
this is a sriov client image I used to confirm that the controller is working, people could use it as a sample.